headers/IMovable.h

```diff
@@ -1,12 +1,12 @@
 #pragma once
 
-class Environment;
+struct Environment;
 
 class IMovable
 {
 public:
 	virtual ~IMovable() = default;
-	virtual void Move(Environment& env) = 0;
+	virtual void Move(Environment* env) = 0;
 
-	int speed{1};
+	int Speed{1};
```

Comment:
var speed become Speed. Why so?

Reply:
Experimental with style ended by the same result as it was, variables names start with lower case.

also this line was removed because speed already have and reasigned in derived class.